config/main.py

```diff
@@ -2272,20 +2272,20 @@ def breakout(ctx, interface_name, mode, verbose, force_remove_dependencies, load
                       remains unchanged to limit the traffic impact """
 
     click.secho("\nAfter running Logic to limit the impact", fg="cyan", underline=True)
-    matched_item = [intf for intf, speed in del_intf_dict.items() if intf in add_intf_dict.keys() and speed == add_intf_dict[intf]]
+    matched_item = [intf for intf, speed in list(del_intf_dict.items()) if intf in list(add_intf_dict.keys()) and speed == add_intf_dict[intf]]
 
     # Remove the interface which remains unchanged from both del_intf_dict and add_intf_dict
-    map(del_intf_dict.pop, matched_item)
-    map(add_intf_dict.pop, matched_item)
+    list(map(del_intf_dict.pop, matched_item))
+    list(map(add_intf_dict.pop, matched_item))
 
     click.secho("\nFinal list of ports to be deleted : \n {} \nFinal list of ports to be added :  \n {}".format(json.dumps(del_intf_dict, indent=4), json.dumps(add_intf_dict, indent=4), fg='green', blink=True))
-    if len(add_intf_dict.keys()) == 0:
+    if len(list(add_intf_dict.keys())) == 0:
```

Comment:
```suggestion
    if len(add_intf_dict) == 0:
```

Reply:
This change was made by the `2to3` tool (see [here](https://www.python.org/dev/peps/pep-0469/#migrating-directly-to-python-3)):

> Note that in many cases d.keys() can be replaced by just d, but the 2to3 migration tool doesn't attempt that replacement.

Plus, keeping `.keys()` makes it clear what the intention is.

Reply:
Changed this to `if not add_intf_dict:`, as it it's just checking for an empty dict.